src/runtime/qurt_hvx.cpp

```diff
@@ -2,6 +2,7 @@
 #include "HalideRuntimeQurt.h"
 #include "printer.h"
 #include "mini_qurt.h"
+#include "hexagon_remote/log.h"
```

Comment:
debug(...) will only show up if you add the `debug` target feature flag. This would make it useful in this case to give feedback to the user (e.g. that their prefetch directive was too big/caused too many prefetches).